lib/face_control/checkers/comments.rb

```diff
@@ -0,0 +1,29 @@
+require 'face_control/comment'
+
+module FaceControl
+  module Checkers
+    class Comments
+
+      def command(filenames)
+        "grep -inEH '(todo|fixme)' #{filenames}"
+      end
+
+      def parse(command_output)
+        comments = []
+        unless command_output.empty?
```

Comment:
Nice check! It's a shame I haven't implemented it myself.

Now I did! See the last commit in `develop`. You can remove it now.